Description: Update rcdom namespace following release 0.25 of 
 html5ever. The API has been changed and the features which used
 to be available under html5ever::rcdom are now in a dedicated crate.
 This patch cope for that change.
Author: Henry-Nicolas Tourneur <debian@nilux.be>
Forwarded: no
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/block.rs
+++ b/src/block.rs
@@ -4,9 +4,9 @@
 
 use super::markup_links;
 
-use html5ever::rcdom::Node;
-use html5ever::rcdom::NodeData;
-use html5ever::rcdom::RcDom;
+use markup5ever_rcdom::Node;
+use markup5ever_rcdom::NodeData;
+use markup5ever_rcdom::RcDom;
 use html5ever::tendril::TendrilSink;
 use html5ever::tree_builder::Attribute;
 use html5ever::tree_builder::TreeBuilderOpts;
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -1,9 +1,9 @@
 use ammonia;
 use failure;
 
-use html5ever::rcdom::Node;
-use html5ever::rcdom::NodeData;
-use html5ever::rcdom::RcDom;
+use markup5ever_rcdom::Node;
+use markup5ever_rcdom::NodeData;
+use markup5ever_rcdom::RcDom;
 use html5ever::tendril::TendrilSink;
 use html5ever::tree_builder::TreeBuilderOpts;
 use html5ever::{parse_document, ParseOpts};
